pegasus/sites.v3/code.org/public/css/dance-landing/dance.css

```
@@ -149,6 +155,10 @@ a:hover {
     background-image: url(/images/fit-1940/dance-hoc/dance_party_landing_main_nophotos.png)
   }
 
+  /* the background image must be flipped horizontally on mobile or else the text will blend into the background */
+  div[dir = 'rtl'] .img-container{
+    background: url(/images/dance-hoc/dance_party_2019_tutorial_hero_flipped.png);
```

```suggestion
    transform: scaleX(-1);
```
Instead of creating a new `png`, have you tried using CSS to flip the background image? I tried this and it looks like it works.
Note that if you accept this change, make sure to go delete the new `png` added to this commit.